SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -447,10 +447,16 @@ void USpatialSender::FlushRPCService()
 	{
 		RPCService->PushOverflowedRPCs();
 
-		for (const SpatialRPCService::UpdateToSend& Update : RPCService->GetRPCsAndAcksToSend())
+		const TArray<SpatialRPCService::UpdateToSend> RPCs = RPCService->GetRPCsAndAcksToSend();
+		for (const SpatialRPCService::UpdateToSend& Update : RPCs)
 		{
 			Connection->SendComponentUpdate(Update.EntityId, &Update.Update);
 		}
+
+		if (RPCs.Num())
+		{
+			Connection->MaybeFlush();
```

Comment:
Ah. Is this the bit which can cause more than 1 flush per tick? What causes a `FlushRPCService` call?

Reply:
Currently any RPC call, although this behavior may change because I don't think it was intended to be that way